accept/src/index.ts

```diff
@@ -3,14 +3,30 @@ import { runAlways } from './always'
 import { runBotReview } from './botReview'
 import { runPr } from './pr'
 
-const defaultMode = 'pr'
+const defaultMode = 'auto'
```

Comment:
Let's not change this (yet) as it might break some repositories which still use this action as part of the build workflow.
```suggestion
const defaultMode = 'pr'
```

Reply:
Actually, this might work as they probably use `push` events only (which results in `always` mode)